pkgs/games/quake3/content/hires.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchzip }:
+
+stdenv.mkDerivation {
+  pname = "quake3hires";
+  version = "2020-05-07"; # Unknown version, used the date of adding the package.
+
+  src = fetchzip {
+    url = https://ioquake3.org/files/xcsv_hires.zip;
```

Comment:
Can this maybe change in the future? Maybe we should upload it to web.archive.org.

Reply:
Good point. Apparently they had a security breach and all downloads are broken https://ioquake3.org/ioquake3/a-new-ioquake3-org/

I still have the unpacked version from 2020-05-07 when I actually packaged this.

I can't get the hash to match with the original archive though, not sure what's best to do here.
I guess the options are:
1. Trust that I'm a trustworthy person with no evil intentions and use the source from archive.org, which I uploaded myself and computed the hash of. Also assume that my machine is not compromised and the binary files in the archive are intact
2. Wait until upstream reuploads the zip, download that, put that in archive.org and use the hash of upstream download

Reply:
I think we need to wait until they reupload or find a link on web.archive.org.

Reply:
I still have the unpacked contents though, just not sure how to recreate the original archive

Reply:
Also you can't upload it to web.archive.org.

Reply:
Ah, I see, sorry, never used it before.
It seems that there actually is a capture from 2020 available, I updated the PR.
Note, that the most recent capture contains a corrupted archive - the contents under URL used match the hash I have locally though